history-of-the-quakers-v4/original/10-book-7-chapter-4.adoc

```diff
@@ -494,6 +497,8 @@ his sentiments on the proper qualification for this service,
 are contained in the following remarks,
 which were found amongst his papers after his decease.
 
+[quote]
```

Comment:
I wasn't sure if this should be a block quote or an embed.

Reply:
I think let's do an embed instead, maybe just leave the type blank. 👍